modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/rocksdb/RocksDbTxStateStorageTest.java

```diff
@@ -43,18 +48,28 @@ public class RocksDbTxStateStorageTest extends AbstractTxStateStorageTest {
     @WorkDirectory
     private Path workDir;
 
+    private final ScheduledExecutorService scheduledExecutor = new ScheduledThreadPoolExecutor(1);
```

Comment:
Not really an issue, but I would suggest to use `Executors.newSingleThreadScheduledExecutor`

Reply:
fix it